### ISSUE TITLE
Support for binary content - OpenAPI v3.1

### ISSUE THREAD

**Issue by tib** [mattpolzin/OpenAPIKit#355]
Hello, 

based on the [OpenAPI 3.1 migration guide](https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0), this is how the users could describe binaries:

OpenAPI v3.0:

```yaml
requestBody:
  content:
    application/octet-stream:
      schema:
        type: string
        format: binary
```

OpenAPI v3.1:

```yaml
requestBody:
  content:
    application/octet-stream: {}
```

My question: is it possible te describe the `{}` somehow using the OpenAPIKit library? I had no luck so far, there is no `.binary` JSONSchema, so I'm not sure how to overcome this issue. Is this feature even supported, or am I missing something? 🤔

Many thanks. 🙏
Tib

**Comment by mattpolzin** [comment]
Yes, this is still possible to represent. For OAS 3.1, use  `contentType: .binary` instead of `format: .binary` but otherwise the same (still a string schema).

**Comment by mattpolzin** [comment]
Woops, I botched the description but you found the answer anyway. Thanks for the correction.
